source/includes/audit-log/_introduction.md

```diff
@@ -61,11 +61,11 @@ Pagination on audit log API allows for retrieving the next page from your query
 1. In the initial request:
     - Set the `paging` [query parameter](https://developers.greenhouse.io/audit-log.html#the-events-object) to `true` to receive a `paging` object with `pit_id` and `next_search_after` in the response
     - Use the `Size` header to specify the number of results you want to receive per page. The minimum value is `100`, and the maximum value is `500`
-1. In subsequent requests:
```

Comment:
While this change is fine, it is not technically needed as markdown will increase the numbers sequentially in the final document.

Reply:
Agreed -- most of these formatting changes were IDE nits that didn't affect the rendering on the page. I resolved them or auto-fixed them to get rid of the yellow Rubymine warning.